public/javascripts/chart/ratingHistory.js

```diff
@@ -97,10 +97,7 @@ lichess.ratingHistoryChart = function(data, singlePerfName) {
               name: serie.name,
               type: 'line',
               dashStyle: dashStyles[i],
```

Comment:
It might be better to do `Solid` dashStyle for all. If the colors on line 65 of this file don't seem different enough, we should consider using a list like this: https://sashamaps.net/docs/tools/20-colors/ (seems like more than enough colors for 99% accessibility). Not sure if some nuance for light vs dark theme.